recipe/meta.yaml

```diff
@@ -36,6 +36,7 @@ requirements:
     - r-rlang >=0.2.0
     - r-storr >=1.1.0
     - r-txtq >=0.1.3
+    - r-knitr
```

Comment:
This most probably needs to be also added to the run section.

Reply:
It's just a "suggests", so should be removed.

Reply:
Removed it.